src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -134,7 +159,7 @@
 
 helps['appservice web source-control config-local-git'] = """
     type: command
-    short-summary: enable local git, you will get a url to clone and later push to the web app
+    short-summary: enable local git. You will get a url to clone and later push to the web app
```

Comment:
The second sentence seems like it should be part of long-summary rather than included in the short-summary. If you used `appservice web source-control -h` would you expect the help for that subcommand to be so verbose? If it were part of the long-summary, then that text would only appear when you do -h on the actual command.